pkg/eval/platform/is-unix.go

```diff
@@ -0,0 +1,6 @@
+// +build aix darwin dragonfly freebsd illumos linux netbsd openbsd solaris
```

Comment:
For consistency with code elsewhere, stick to `!windows,!plan9,!js`.

Reply:
Okay